common/version.go

```diff
@@ -0,0 +1,53 @@
+package common
```

Comment:
Can't we put this code in the `main.go` file? So we'll be able to use GoReleaser's default ldflags: https://goreleaser.com/environment/#using-the-mainversion

Reply:
The problem if we moved this code in the `main.go` is that we will have to use a global variable from the `main` package in the `cmd` package. The latter being used to define CLI commands (and manage the output version of the `--version` flag).

Moreover, if this PR is accepted, I've planned to add a metadata metric to expose those information in Prometheus. So this structure will be also used in the `hub` package thought the metrics collector.

That's why, I decided to add the code in a `common` package.

Reply:
Ok let's keep this as is then!